lib/dal/tests/integration_test/schema/variant/view.rs

```diff
@@ -20,3 +21,21 @@ async fn list_schema_variant_views(ctx: &DalContext) {
         views.len()               // actual
     )
 }
+
+#[test]
+async fn list_schema_variant_definition_views(ctx: &DalContext) {
+    let schema_variant_ids = SchemaVariant::list_ids(ctx)
+        .await
+        .expect("could not list schema variants");
+
+    // We are going to ensure that we get a default schema variant
+    // for each schema variant
+    let views = SchemaVariantDefinitionView::list(ctx)
+        .await
+        .expect("could not list schema variant views");
+
+    assert_eq!(
+        schema_variant_ids.len(), // expected
+        views.len()               // actual
+    )
```

Comment:
I think this will fail if a `Schema` has more than one variant

Reply:
Which is EXACTLY what we want :) It's wju I added it - because there always has to be a default for a variant :)